package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
```

Comment:
Poderíamos usar o parâmetro `-E` de exact na instalação de packages, dessa forma evitando instalar versões aproximadas.

> Dependencies saved to package.json will be configured with an exact version rather than using npm's default semver range operator.

Referência:

- https://yarnpkg.com/cli/add#options-E%2C-exact
- https://docs.npmjs.com/cli/v7/commands/npm-install#save-exact